web_poet/page_inputs/client.py

```diff
@@ -265,3 +265,18 @@ async def batch_execute(
     def get_saved_responses(self) -> Iterable[_SavedResponseData]:
         """Return saved requests and responses."""
         return self._saved_responses.values()
+
+
+def _http_status_name(status: int):
+    """
+    >>> _http_status_name(200)
+    'OK'
+    >>> _http_status_name(404)
+    'NOT_FOUND'
```

Comment:
There is an option to use "phrase" instead of "name", to make it "Not Found" instead of "NOT_FOUND"; I haven't changed it in this PR.